crates/vault-registry/src/lib.rs

```diff
@@ -737,7 +826,11 @@ impl<T: Config> Pallet<T> {
                 );
                 Self::try_deposit_collateral(account, amount)?;
             }
-            CurrencySource::Griefing(_) | CurrencySource::ReservedBalance(_) | CurrencySource::LiquidationVault => {
+            CurrencySource::Griefing(_) | CurrencySource::ReservedBalance(_) => {
```

Comment:
In relation to the comment above, it would only make sense to decrease the total backing collateral if it is increased here.